package.json

```diff
@@ -30,9 +31,9 @@
     "@babel/preset-env": "^7.16.5",
     "@babel/preset-react": "^7.16.5",
     "babel-loader": "^8.2.3",
-    "webpack": "^5.65.0",
-    "webpack-cli": "^4.9.1",
     "jest": "^27.4.5",
-    "nodemon": "^2.0.15"
+    "nodemon": "^2.0.15",
+    "webpack": "^5.65.0",
+    "webpack-cli": "^4.9.1"
```

Comment:
Cool! I noticed there's also a package called ['react-bootstrap'](https://react-bootstrap.github.io/getting-started/introduction/). Is there a reason why you chose vanilla bootstrap instead?